recipes/snap/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: snap
+  version: '2013_11_29'
+
+source:
+  fn: snap-2013-11-29.tar.gz
+  url: http://korflab.ucdavis.edu/Software/snap-2013-11-29.tar.gz
+  #url: https://github.com/amplab/snap/archive/v1.0beta.18.tar.gz
+
+build:
+  number: 0
+
+requirements:
+  build:
```

Comment:
Sorry to not be clear. We need to specify both gcc and llvm depending on the OS, like here: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/openbabel/2.3.2/meta.yaml#L20